src/components/comments/index.test.ts

```diff
@@ -12,30 +16,48 @@ describe('CommentsComponent', () => {
   });
 
   it('should have a name property', () => {
-    expect((commentsComponent as any).name).toEqual('Comments');
+    expect(commentsComponent['name']).toBe('Comments');
   });
 
   it('should have a logger property', () => {
-    expect((commentsComponent as any).logger).toBeDefined();
+    expect(commentsComponent['logger']).toBeDefined();
   });
 
   it('should have an element property', () => {
-    expect((commentsComponent as any).element).toBeUndefined();
+    expect(commentsComponent['element']).toBeUndefined();
   });
 
   it('should create a new element when start() is called', () => {
-    (commentsComponent as any).start();
-    expect((commentsComponent as any).element).toBeDefined();
+    commentsComponent.attach({
+      realtime: ABLY_REALTIME_MOCK,
+      localParticipant: MOCK_LOCAL_PARTICIPANT,
+      group: MOCK_GROUP,
+      config: MOCK_CONFIG,
+    });
+    commentsComponent.detach();
+    expect(commentsComponent['element']).toBeDefined();
```

Comment:
When detach is called this element shoud be destroyed and the variable should be undefined, right? 

```suggestion
    commentsComponent.detach();
    expect(commentsComponent['element']).toBeDefined();
```